src/chi/utilities/anchor.scss

```diff
@@ -0,0 +1,51 @@
+@import 'mixins';
+@import 'variables';
+
+.chi {
+
+  .-anchor {
+    &::before {
+      content: '';
+      display: block;
+      height: 5rem;
```

Comment:
@davidojedacl for what is it necessary?

Reply:
It is necessary because we have a header bar of 4 rem tall. And we need to put a margin of 5rem at least to watch the title. If not, the header cover the title.
<img width="1419" alt="screen shot 2019-02-19 at 4 18 28 pm" src="https://user-images.githubusercontent.com/43439742/53026103-8d165080-3462-11e9-9435-4a2ff2865e0b.png">

Reply:
@davidojedacl these properties must be in chi-docs.scss and can not be as part of the general anchor.scss, as we don't want to make it generic